code/src/components/Checkbox.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { 
+  Dot,
+  CheckBox,
+  CheckboxContainer,
+  StyledCheckBox
+} from './styled/checkbox'
+
+export const Checkbox = ({ isChecked, onChangeHandler }) => {
+  const [checked, setChecked] =useState(isChecked)
+
+  const handleOnChange = event => {
+    onChangeHandler()
+    setChecked(event.target.checked)
+  }
+
+  const Checkbox = ({ className, checked, ...props }) => (
+    <CheckboxContainer>
+      <CheckBox checked={checked} {...props}></CheckBox>
+      <StyledCheckBox checked={checked}>
+        <Dot viewBox='0 0 24 24'>
+          <circle cx='12' cy='12' r='4' stroke='none' fill='#F9EFE5'></circle>
+        </Dot>
+      </StyledCheckBox>
+    </CheckboxContainer>
```

Comment:
Super nice you made your own custom checkbox! I would have changed the colours slightly just from an UX point of view, as for me when I load the page they're are all green, which for me means immediately "Done!" and it's the opposite on your appo :p